tv_apps/tv_shared/components/smart-dialog/style.css

```diff
@@ -0,0 +1,99 @@
+/**
+ * Smart-Dialog
+ *   - State:
+ *       1. closed
+ *       2. opening
+ *       3. opened - content visible
+ *       4. closing
+ * Default scale: 1920x1080 (2*sqrt(960*960 + 1080*1080) / 20 = 144.5)
+ */
+
+smart-dialog {
+  position: fixed;
+  z-index: 2147483647;
+  height: 2rem;
+  width: 2rem;
+  top: calc(100% - 1rem);
+  left: calc(50% - 1rem);
+  background-color: rgba(0, 0, 0, 0.9);
+  transition: none;
+  transform: scale(20);
+}
+
+smart-dialog:before {
+  content: attr(title);
+  display: block;
+  margin-top: 5rem;
+  margin-left: 7.6rem;
+  font-family: "Fira Sans";
+  font-size: 5.4rem;
+  font-style: italic;
+  color: #ffffff;
+}
+
+smart-dialog > * {
+  display: none;
+}
+
+smart-dialog.closed {
+  visibility: hidden;
+}
+
+smart-dialog.opened {
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  transform: scale(1);
+}
+
+smart-dialog.opened > * {
```

Comment:
Since we are creating a container, we shouldn't override the `display` attribute of child to `block`. If we want to hide it, please use the following rule:
```
smart-dialog.closed > *,
smart-dialog.closing > *,
smart-dialog.opening > * {
  display: none;
}
```